examples/xgboost/README.md

```diff
@@ -1,2 +0,0 @@
-## Example Notebooks
```

Comment:
Was removing this intentional?

Reply:
There is no docs for XGBoost here currently, so I thought no point in keeping just a read me with external link.

Reply:
We do have a doc page for XGBoost though, I think it would be nice if we have this page just to be clearer.